### PR TITLE
Abhilaksh

### PR DESCRIPTION
**Pull request by abhilakshm**
Updated the get_user function according to Update_user

### CODE REVIEW

File: src/Database/database.py [hunk 1]
```diff
@@ -45,11 +45,18 @@ def get_user_data(user_id, table):
...
+                module_id = item.get('moduleId')
                 module_data = item.get('moduleData', {})
-                combined_user_data[item['moduleId']] = module_data
-
+                module_name = item.get('moduleName', '')
+                
+                if module_id not in combined_user_data:
+                    combined_user_data[module_id] = {"moduleData": module_data, "moduleName": module_name}
+                else:
+                    combined_user_data[module_id]["moduleData"].update(module_data)
```
Comment (kaivan-s) [inline src/Database/database.py:57]: What is this logic for ?

File: src/app.py [hunk 2]
```diff
@@ -72,4 +72,4 @@ def update_user():
     try:
         app.run(host="0.0.0.0", debug=True)
     except NoCredentialsError:
-        print("Error: AWS credentials not found. Make sure to set up your credentials.")
+        print("Error: AWS credentials not found. Make sure to set up your credentials.")
```
Comment (kaivan-s) [inline src/app.py:75]: raise exception as above instead of printing.

**Review by kaivan-s — CHANGES_REQUESTED** [review]
Added a few comments. Please check.

**Review by kaivan-s — COMMENTED** [review]
Have you verified the API after making the changes? Is everything working fine?

**Comment by abhilakshm** [comment]
> Have you verified the API after making the changes? Is everything working fine?

>Yes it is working fine

**Review by kaivan-s — APPROVED** [review]

**Pull request merged**